apollo-compiler/api/apollo-compiler.api

```diff
@@ -1969,18 +1980,20 @@ public abstract class com/apollographql/apollo3/compiler/ir/IrType {
 }
 
 public final class com/apollographql/apollo3/compiler/ir/IrUnion : com/apollographql/apollo3/compiler/ir/IrSchemaType {
-	public fun <init> (Ljava/lang/String;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;)V
+	public fun <init> (Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;)V
 	public final fun component1 ()Ljava/lang/String;
-	public final fun component2 ()Ljava/util/List;
-	public final fun component3 ()Ljava/lang/String;
+	public final fun component2 ()Ljava/lang/String;
+	public final fun component3 ()Ljava/util/List;
 	public final fun component4 ()Ljava/lang/String;
-	public final fun copy (Ljava/lang/String;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;)Lcom/apollographql/apollo3/compiler/ir/IrUnion;
-	public static synthetic fun copy$default (Lcom/apollographql/apollo3/compiler/ir/IrUnion;Ljava/lang/String;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;ILjava/lang/Object;)Lcom/apollographql/apollo3/compiler/ir/IrUnion;
+	public final fun component5 ()Ljava/lang/String;
+	public final fun copy (Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;)Lcom/apollographql/apollo3/compiler/ir/IrUnion;
+	public static synthetic fun copy$default (Lcom/apollographql/apollo3/compiler/ir/IrUnion;Ljava/lang/String;Ljava/lang/String;Ljava/util/List;Ljava/lang/String;Ljava/lang/String;ILjava/lang/Object;)Lcom/apollographql/apollo3/compiler/ir/IrUnion;
 	public fun equals (Ljava/lang/Object;)Z
 	public final fun getDeprecationReason ()Ljava/lang/String;
 	public final fun getDescription ()Ljava/lang/String;
 	public final fun getMembers ()Ljava/util/List;
 	public fun getName ()Ljava/lang/String;
+	public fun getTargetName ()Ljava/lang/String;
 	public fun hashCode ()I
 	public fun toString ()Ljava/lang/String;
```

Comment:
The compiler doesn't like it because they are part of the public API of builders (`CustomScalarBuilder`, `EnumAsEnumBuilder`, etc.). I could make these internal too?

Reply:
Yep, the public API of `apollo-compiler` is just `ApolloCompiler` and `Options` at the moment